src/FondOfSpryker/Yves/GoogleTagManagerStoreConnector/Model/GoogleTagManagerStoreConnectorModel.php

```diff
@@ -31,44 +31,54 @@ public function __construct(
     }
 
     /**
+     * @param string $page
+     * @param array $twigVariableBag
+     * @param array $variableList
+     *
      * @return array
      */
-    public function getCurrency(): array
+    public function getCurrency(string $page, array $twigVariableBag, array $variableList): array
```

Comment:
Unused variables

Reply:
Probably due to expander interface.. guess that's fine then, nvm ( :

Reply:
jepp :) we are not using any parameters from outside to get the currency. but it is also possible that some else want to inject parameters i.e. from yves.